Update travel.md (add private car reimbursement) (#439)
madetech/handbook: guides/compensation/expenses/travel.md

If you incur travel expenses as part of your work at Made Tech, the business will reimburse your standard class travel costs.

Some examples of work travel you can expense:

- Public transport / taxi to a client site for a meeting
- Taxi home after working late
- Travelling to work at a client site outside of the region of your Made Tech office

## Using your own car

> You can use your private car where you cannot or it's not practical to use public transportation to travel for business. The current reimbursement rate as prescribed by HMRC is 45p per mile (up to 10k miles).

Reviewer: I think a link to the HMRC web page would be good here. I assume it's the following link - https://www.gov.uk/expenses-and-benefits-business-travel-mileage/rules-for-tax

If we do add the link, maybe the heading for this section can be updated to say "Using your own vehicle" instead(?).

Author: Great idea. Updated.

When claiming your mileage, please use a journey planner such as Google Maps to get the round-trip distance and include a screenshot when expensing. 

> To allow MadeTech to claim the VAT on the journey’s fuel element, please can you provide the following:

Reviewer: ```suggestion
To allow Made Tech to claim the VAT on the journey’s fuel element, please provide the following:
```

- A petrol receipt (showing VAT) dated max 2 months before the journey.
- Add to the expense description the engine size of the car and if it's petrol or diesel.

This part is optional and the expense total should always be miles*applicable reimbursement rate.

> If using your own car, you must ensure that adequate insurance and breakdown cover is in place. MadeTech will not assume any responsibility for breakdowns, fines, accidents or any damage. If unsure of any of the above, please contact Finance for additional guidance.

Reviewer: ```suggestion
If using your own car, you must ensure that adequate insurance and breakdown cover is in place. Made Tech will not assume any responsibility for breakdowns, fines, accidents or any damage. If unsure of any of the above, please contact the Finance team for additional guidance.
```

## Travel that is not covered

> Where you're employed as a staff member in a Made Tech office, your regular travel to that office or to a client site within that metropolitan area is not be covered.

Reviewer: ```suggestion
Where you're employed as a staff member in a Made Tech office, your regular travel to that office, or to a client site, within that metropolitan area is not covered.
```
